ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPageVulnerabilities.tsx

```diff
@@ -284,7 +290,7 @@ function DeploymentPageVulnerabilities({
                 <Divider />
                 <div className="pf-v5-u-flex-grow-1 pf-v5-u-background-color-100">
                     <div className="pf-v5-u-p-lg">
-                        <Split className="pf-v5-u-pb-lg pf-v5-u-align-items-baseline">
+                        <Split hasGutter className="pf-v5-u-pb-lg pf-v5-u-align-items-baseline">
```

Comment:
Off the subject of this contribution, change brings a question to mind about pro and con of alternatives?
* `Flex`
* `Toolbar`

Reply:
Yes this deserves some more investigation as some sections of VM use `Toolbar` for the equivalent element and this uses `Split` peppered with utility classes. Unifying the markup was a tech debt task from a few releases ago that hasn't quite been prioritized yet.